### ISSUE TITLE
Wiki/wiky typo

### ISSUE THREAD

**Issue by zachzundel** [SynBioHub/synbiohub#433]
The wiki is referred to as `wiky` many places in the code, they should all be `wiki` for consistency and correctness

**Comment by ethanwillis** [comment]
https://github.com/facebook/codemod

**Comment by cjmyers** [comment]
It is "wiky" because the library being used is called wiky.  So, not sure we want to change this afterall, unless there are some "wiki" to change to "wiky".

**Comment by udp** [comment]
Really we should parse the iGEM/mediawiki wiki syntax properly rather than using the wiky plugin. We currently only support a subset of what's available on the parts registry.

**Comment by cjmyers** [comment]
Agreed.  How standard is wiki syntax?  I was reading online that it is not very standard.  

I was playing around a bit with dropping iGEM wiki source into our wiki box and a lot of things do not translate correct or at all.  Some even causes errors.  I was also trying to render the external images, and I eventually got it to work once I determined where the image resides, but it is tricky to determine this.  It seems the directory structure for the image is obvious from the way the image is linked.

I was doing this because I was thinking if we improve our wiki render that we could use the raw fetch to render their pages in real-time rather than doing a scrape.  This would allow us to always present the iGEM registry pages as the currently appear.  There is clearly though some work to make this actually a reality.
